upload-s3-artifact/action.sh

```diff
@@ -40,13 +42,14 @@ main() {
     echo "No file or directory at path '$INPUT_TARGET_PATH'"
     exit 1
   fi
+  aws_s3_key_prefix="${INPUT_AWS_S3_KEY_PREFIX:-""}"
```

Comment:
Do we need this? The default value is already set through `INPUT_AWS_S3_KEY_PREFIX`

Reply:
No, it can be removed, I put it in before I added the default!